src/lib/constants.js

```diff
@@ -24,6 +24,10 @@ export const HEADERS = {
   TEXT_PLAIN: 'text/plain',
   TEXT_HTML: 'text/html',
   FORWARDED_PROTO: 'X-Forwarded-Proto',
+  XFRAMES_OPTIONS: 'X-Frame-Options',
```

Comment:
Maybe be consistent and name it `FRAME_OPTIONS`.

Reply:
`Feature-Policy` might be also helpful (configurable in the backend) and HSTS.

Reply:
Let me read about that one

Reply:
https://developers.google.com/web/updates/2018/06/feature-policy 👍

Reply:
I won't use `Feature-Police` since it basically needs a set of features to allow and disallow and we don't use any.